src/components/features/ProgressTrack/ProgressTrack.tsx

```diff
@@ -52,6 +53,8 @@ export interface BaseProgressTrackProps {
   onEdit?: () => void;
   hideDifficultyLabel?: boolean;
   hideRollButton?: boolean;
+  useSmallRollButton?: boolean;
+  useMaxRoll?: boolean;
```

Comment:
nit: I try to avoid prefixing props with `use` to prevent any confusion with hooks.

Reply:
good point